### DISCUSSION TITLE
Missing value in syncing translation repo

### DISCUSSION THREAD

**Discussion by dngduy132** [WeblateOrg/weblate#11409] [Q&A]
Hello, 
The issue happened when I updated some lines in resource bundle files for all 4 languages (FR, DE, IT, EN), after committing the changes to Bitbucket, just FR and EN synced to Weblate, the other language did not. 
Is there any solution other than deleting and creating the component again to get the latest value?
Thanks and best regards.

**Comment by nijel** [comment]
You can use sync option in the repository management.

**Reply by dngduy132** [reply]
I see that the synchronize option here is to force writing all translations to the Weblate repository, I don't know if this option is what you mean. 
![image](https://github.com/WeblateOrg/weblate/assets/92735527/17d45e8d-266f-45be-b26f-c13b4f7eee6a)

what I want to mention is that Weblate didn't get the latest value in the bundle file, for example with DE language, the bundle file is 
![image](https://github.com/WeblateOrg/weblate/assets/92735527/3f223858-7c52-4148-b038-0eaceae96741)
but Weblate showed the old value before I updated the file 
![image](https://github.com/WeblateOrg/weblate/assets/92735527/c39cd722-2789-4d83-afd7-6848f495339f)
as you can see, FR and EN are synced but not DE and IT
